src/platform_impl/windows/window.rs

```diff
@@ -771,9 +771,13 @@ impl CoreWindow for Window {
             // Change video mode if we're transitioning to or from exclusive
             // fullscreen
             match (&old_fullscreen, &fullscreen) {
-                (_, Some(Fullscreen::Exclusive(video_mode))) => {
-                    let monitor = video_mode.monitor();
+                (_, Some(Fullscreen::Exclusive(monitor, video_mode))) => {
                     let monitor_info = monitor::get_monitor_info(monitor.hmonitor()).unwrap();
+                    let video_mode =
+                        match monitor.video_mode_handles().find(|mode| &mode.mode == video_mode) {
+                            Some(monitor) => monitor,
+                            None => return,
```

Comment:
Perhaps this is planned for a future change, but are we missing any feedback to the user here that the passed `VideoMode` isn't valid and wasn't applied?

I expect that `video_mode_handles()` _might_ change during app runtime.

Reply:
Video modes usually don't change, unless monitor is reconfigured. In general though, there's no guarantee that the API will work, though, on windows it likely should generally work.

Reply:
I'd add an error for that, but once we have monitor handle changed.